docs/proposals/operator-hub.md

```diff
@@ -0,0 +1,197 @@
+mariadb-operator-0.1.3-6      ephemeral,persistent     OperatorHub
+```
+### Only Service Catalog is enabled
+```
+$ odo catalog list services
+NAME                          PLANS                    PROVISIONER
+mariadb-ephemeral             default                  ServiceCatalog
+mongodb-ephemeral             default                  ServiceCatalog
+mysql-ephemeral               default                  ServiceCatalog
+```
+
+### Only Operator Hub is enabled
+```
+$ odo catalog list services
+NAME                          PLANS                    PROVISIONER
+mongodb-enterprise.v1.2.4     prod                     OperatorHub
+mariadb-operator-0.1.3-6      ephemeral,persistent     OperatorHub
+```
+
+When working on OpenShift, we can get a list of enabled Operators in a
+namespace (project) by doing `oc get csvs`. 
+
+### Kubernetes
+Listing Operators in Kubernetes might involve some extra tinkering. Operator
+Hub is not pre-installed on a minikube cluster. In absence of any Operators,
+`kubectl get crds` still works and shows No resource found  but `kubectl get
+csvs` doesn’t work since there’s no API resources with that name in Kubernetes.
+
+One needs to manually install OLM from its
+[repo](https://github.com/operator-framework/operator-lifecycle-manager/tree/00eab85df0fd570891754cc743bc3d5831e9dd62/deploy/upstream/quickstart)
+to make ClusterServiceVersion objects available in Kubernetes.
+
+## Link components with operands
+[GitHub issue](https://github.com/openshift/odo/issues/2463)
+
+In its current form, linking a component with another component/service
+involves making the Secrets (environment variables) of the component/service
+being linked available to the component it is linked with. For example, if
+`component A` needs to be connected to `service A`  then the environment variables
+of `service A` will be made available in `component A`.
+
+Although Service Binding Operator could provide an easy mechanism to link the
+components with operands created from an operator, we intend to explore linking
+the two without adding the dependency on this operator. This should help us be
+agnostic of backend cluster being OpenShift or Kubernetes.
+
+## Listing the services
+[GitHub issue](https://github.com/openshift/odo/issues/2479)
+
+At the moment, we use `odo service list --app <app-name> --project
+<project-name>` to list the services in a given application and project. We
+could augment this to have the information about the services created from the
+Operators. To make a distinction between the source of the services in the
+cluster, we could add a column to the output just like we plan to do for
+listing available/installed operators via `odo catalog list services`.
+
+```sh
+$ odo services list
+NAME                          PLANS                    PROVISIONER
+mongodb-enterprise.v1.2.4     prod                     OperatorHub
+mariadb-operator-0.1.3-6      ephemeral,persistent     ServiceCatalog
```

Comment:
The command lists service instances. Each service instance uses only one plan that was specified during the service creation

Reply:
Yes, this section is about listing the service instances or the services/operands created using Operators. I pasted the wrong output.